packages/common/utils/item.js

```diff
@@ -52,7 +52,7 @@ export const generateSystemItemName = (entity, id) => {
 };
 
 export const generateSystemItemEmail = entityName => {
-  return `systems+${entityName}@${DOMAIN_HOSTNAME || getHostName()}`;
+  return `systems+${entityName}@${DOMAIN_HOSTNAME || getHostName()}.com`;
```

Comment:
(!) ALARM (!)
@beshkenadze removed `.com` 5 days ago in [pr](https://github.com/caesar-team/caesar-web-client/pull/523/files#diff-f36b1f5dcf3b12b3094535e16cfe0366fb2791616077decea55483dc23e73a6e).
Please, determine what should be here.

Reply:
Accidently commited

Reply:
Do not put the third level of domain by hand. If you want to change the host name you have to use the env `DOMAIN_HOSTNAME`